website/content/docs/commands/operator/raft.mdx

```diff
@@ -144,6 +144,10 @@ Usage: vault operator raft remove-peer <server_id>
 	  $ vault operator raft remove-peer node1
 ```
 
+<Note>
+  Once a node is removed, its Raft data needs to be removed before it may be rejoined or re-added back into a cluster. This requires a restart of the Vault process on the removed node.
```

Comment:
```suggestion
  Once a node is removed, its Raft data needs to be deleted before it may be joined back into an existing cluster. This requires shutting down the Vault process, deleting the data, then restarting the Vault process on the removed node.
```

Reply:
Thank you @raskchanky, that reads easier and provides clarity! 
LGTM!